hub/Dockerfile

```diff
@@ -2,29 +2,25 @@ ARG baseImage="unityci/base"
 FROM $baseImage
 
 # Hub dependencies
-RUN apt-get -q update \
- && apt-get -q install -y --no-install-recommends --allow-downgrades zenity \
- && apt-get clean
+RUN sudo apt-get -q update \
+ && sudo apt-get -q install -y --no-install-recommends --allow-downgrades zenity \
+ && sudo apt-get clean
 
 # Download & extract AppImage
 RUN wget --no-verbose -O /tmp/UnityHub.AppImage "https://public-cdn.cloud.unity3d.com/hub/prod/UnityHub.AppImage" \
  && chmod +x /tmp/UnityHub.AppImage \
  && cd /tmp \
  && /tmp/UnityHub.AppImage --appimage-extract \
- && cp -R /tmp/squashfs-root/* / \
- && rm -rf /tmp/squashfs-root /tmp/UnityHub.AppImage \
  && mkdir -p "$UNITY_PATH" \
- && mv /AppRun /opt/unity/UnityHub
+ && cp -R /tmp/squashfs-root/* $UNITY_PATH \
+ && rm -rf /tmp/squashfs-root /tmp/UnityHub.AppImage \
+ && mv $UNITY_PATH/AppRun $UNITY_PATH/UnityHub
 
 # Alias to "unity-hub" with default params
-RUN echo '#!/bin/bash\nxvfb-run -ae /dev/stdout /opt/unity/UnityHub --no-sandbox --headless "$@"' > /usr/bin/unity-hub \
+RUN echo '#!/bin/bash\nAPPIMAGE_SILENT_INSTALL=true APPDIR='$UNITY_PATH' xvfb-run -ae /dev/stdout '$UNITY_PATH'/UnityHub --no-sandbox --headless "$@"' > /usr/bin/unity-hub \
```

Comment:
Quote style that we're using everywhere is 

```bash
"PARAM=$SOME_VAR"
```
or 
```bash
"${SOME_VAR}suffix"
```
or
```bash
"${UNITY_PATH}/editors/"
```

single quotes is fine too. 

just not 

```bash
'$UNITY_PATH'/UnityHub
```

Especially because it's using quotes around _part of the string_, within a string that's _already quoted in single quotes_ as well.

Reply:
The problem here is that we're echoing a long string to a text file, and we need ARGs to be passed in...
Maybe I'm confused as to the string interpolation syntax interactions between docker and bash, but I'll take a look

I wouldn't mind a recommendation TBH

Reply:
you can just quote args like the last arg in that sentence: `"$@"`

so `'$UNITY_PATH'/UnityHub` becomes `"$UNITY_PATH/UnityHub"` 

and `APPDIR='$UNITY_PATH'` becomes `"APPDIR=$UNITY_PATH"`

Reply:
This doesn't work, because we're in a single-quote string

In the case of the `"$@" ` we actually want `"$@"` in the body of our `unity-hub` script
Those single quotes are actually closing and opening, not opening and closing

But it turns out they're not needed at all

Reply:
Let's not touch existing quotes. They originate from some original script.

I don't see how they're closing and opening instead or vice versa by the way.

I guess the newly added quotes are not necessary